testsuite/CUDA/cooperative_groups/fail/divergence_grid_group/kernel.cu

```diff
@@ -0,0 +1,22 @@
+//xfail:NOT_ALL_VERIFIED
```

Comment:
How does this test fail? With a data race or barrier divergence?

Reply:
This fails because of a barrier divergence. Only the even threads will reach **synchronize(g);** and the odd threads won't which results in a barrier divergence. In this example, the synchronization is happening for a grid_group and in the next example, the synchronization is happening for a thread_group.